util/gh-pages/script.js

```diff
@@ -537,6 +542,16 @@ function getQueryVariable(variable) {
     }
 }
 
+function storeValue(settingName, value) {
+    try {
+        localStorage.setItem(`clippy-lint-list-${settingName}`, value);
+    } catch (e) { }
```

Comment:
It's not because of quota but because `localStorage` might not be supported.

Reply:
Letting it throw would still be fine, we'd want to see the error in the console

Do we support any browsers where that's the case? I know way back when safari in private browsing behaved like that but that was fixed a long time ago also

It could happen on `file:` URLs but the site already doesn't work there due to the JSON file

Reply:
No it should definitely not fail. Not saving a setting is ok, crashing the script is not.

You can see the list of (not) supported platforms [here](https://caniuse.com/?search=localstorage). Basically, small web browsers used on some specific embed/mobile devices.

Reply:
Throwing in an event handler doesn't stop the script, it doesn't even unregister the handler

The grey is unknown not unsupported, the MDN sourced data isn't that thorough. [A related entry](https://caniuse.com/namevalue-storage) from caniuse shows all the small ones support it with the exception of Opera Mini, but that's not relevant because it doesn't run JS